images/win/scripts/Tests/WindowsFeatures.Tests.ps1

```diff
@@ -0,0 +1,25 @@
+Describe "WindowsNETFeatures" {
```

Comment:
Rename to more obvious  name
WindowsNETFeatures -> WindowsFeatures

Reply:
Fixed